core/controllers/question_editor.py

```diff
@@ -130,6 +130,11 @@ def put(self, question_id):
             question_domain.QuestionChange(change)
             for change in self.payload.get('change_list')
         ]
+
+        for change in change_list:
+            if change.cmd == question_domain.CMD_ADD_QUESTION_FROM_SUGGESTION:
```

Comment:
I think we can make the name more generic: CMD_CREATE_NEW_FULLY_SPECIFIED_QUESTION

Reply:
Done